addons/frag/initSettings.inc.sqf

```diff
@@ -3,38 +3,60 @@ private _category = format ["ACE %1", localize LSTRING(Module_DisplayName)];
 [
     QGVAR(enabled), "CHECKBOX",
     [LSTRING(EnableFrag), LSTRING(EnableFrag_Desc)],
-    _category,
+    [_category, LSTRING(Frag)],
     true,
     1
 ] call CBA_fnc_addSetting;
 
 [
     QGVAR(spallEnabled), "CHECKBOX",
     [LSTRING(EnableSpall), LSTRING(EnableSpall_Desc)],
-    _category,
+    [_category, LSTRING(Spall)],
     false,
     1
 ] call CBA_fnc_addSetting;
+
 [
     QGVAR(reflectionsEnabled), "CHECKBOX",
     [LSTRING(EnableReflections), LSTRING(EnableReflections_Desc)],
-    _category,
+    [_category, LSTRING(Reflections)],
     false,
     1
 ] call CBA_fnc_addSetting;
 
 [
-    QGVAR(maxTrack), "SLIDER",
-    [LSTRING(MaxTrack), LSTRING(MaxTrack_Desc)],
-    _category,
-    [0, 50, 10, -1],
+    QGVAR(fragSimComplexity), "LIST",
+     [LSTRING(FragMode), LSTRING(FragMode_Desc)],
```

Comment:
Might as well bring all setting up to standard:

```suggestion
    QGVAR(fragSimComplexity),
    "LIST",
    [LSTRING(FragMode), LSTRING(FragMode_Desc)],
```

Reply:
Should be good as of https://github.com/acemod/ACE3/pull/9728/commits/a4f2ad45d8a95973fbbc65a1d38f9f2dc57b44f6